modules/Registration/components/teacher/TeacherRegistrationForm.tsx

```diff
@@ -0,0 +1,262 @@
+import React from "react";
+
+import { Button, Group, Box, Flex, Text, Grid, Anchor } from "@mantine/core";
+import { useForm } from "react-hook-form";
+import { MultiSelect, TextInput, Select, PasswordInput } from "react-hook-form-mantine";
+
+import {
+  TeacherFormValues,
+  TeacherRegistrationSchemaResolver,
+} from "./TeacherRegistrationForm.helpers";
+import { ITeacherRegistrationFormProps } from "./TeacherRegistrationForm.type";
+
+export const Subjects = ["Physics", "Chemistry", "Math", "Geography", "Communication"];
+
+const TeacherRegistrationForm: React.FC<ITeacherRegistrationFormProps> = ({ onSubmit }) => {
+  const {
+    formState: { errors },
+    handleSubmit,
+    control,
+    reset,
+  } = useForm<TeacherFormValues>({
+    defaultValues: {
+      code: "",
+      email: "",
+      password: "",
+      confirmPassword: "",
+      majorSubject: "",
+      highestEducationLevel: "",
+      subjectsToTeach: [],
+      gender: "",
+      firstName: "",
+      lastName: "",
+    },
```

Comment:
Let's export this from a constants file.

`export const TeacherRegistrationFormInitialValues`